build-system/pr-check/utils.js

```diff
@@ -210,6 +207,22 @@ function timedExec(cmd, fileName = 'utils.js') {
   return p;
 }
 
+/**
+ * Executes the provided command and times it. Errors, if any, are returned.
+ * @param {string} cmd
+ * @param {string} fileName
+ * @return {<Object>} Process info.
+ */
+function timedExecWithError(cmd, fileName = 'utils.js') {
+  const startTime = startTimer(cmd, fileName);
+  const p = exec(cmd, {'stdio': ['inherit', 'inherit', 'pipe']});
+  stopTimer(cmd, fileName, startTime);
+  if (p.stderr.length > 0) {
+    p.error = new Error(p.stderr.toString());
+  }
+  return p;
+}
```

Comment:
Would it make sense to add a function called `execWithError` to `build-system/exec.js`, and use it here?